modules/cluster-logging-collector-external.adoc

```diff
@@ -35,36 +35,51 @@ To send a copy of Fluentd logs to an external log aggregator:
 
 . Edit the `secure-forward.conf` section of the Fluentd configuration map:
 +
-.Sample `secure-forward.conf` section
 ----
 $ oc edit configmap/fluentd -n openshift-logging
 
-<store>
-  @type forward
-  <server> <1>
+  <store>
+  @type secure_forward
+
+  self_hostname ${hostname} <1>
+  shared_key <SECRET_STRING> <2>
+
+  transport tls <3>
+  tls_verify_hostname true <4>
+
+  tls_cert_path /path/for/certificate/ca_cert.pem <5>
+
+...
+
+  <server> <6>
     name externalserver1
     host 192.168.1.1
     port 24224
   </server>
-  <server> <1>
+  <server> <6>
     name externalserver2
     host 192.168.1.2
     port 24224
   </server>
-</store>
+  </store>
 ----
-<1> Enter the name, host, and port for your external Fluentd server.
 
-. Add certificates to be used in `secure-forward.conf` to the existing
+<1> Specify the default value of the auto-generated certificate common name (CN).
+<2> Enter the Shared key between nodes
+<3> Specify `true` to enable TLS validation.
+<4> Set to `true` to verify the server cert hostname. Set false to ignore server cert hostname.
+<5> Specify the path to private CA certificate file.
+<6> Enter the name, host, and port for your external Fluentd server.
+If you specify two or more servers, `out_secure_forward` uses these server nodes in a round-robin order.
+
+. Add certificates to be used in `forward.conf` to the existing
 secret that is mounted on the Fluentd pods. The `your_ca_cert` and
-`your_private_key` values must match what is specified in `secure-forward.conf`
+`your_private_key` values must match what is specified in `forward.conf`
 in the `fluentd` ConfigMap:
 +
 ----
-$ oc patch secrets/fluentd --type=json \
-  --patch "[{'op':'add','path':'/data/your_ca_cert','value':'$(base64 /path/to/your_ca_cert.pem)'}]"
-$ oc patch secrets/fluentd --type=json \
-  --patch "[{'op':'add','path':'/data/your_private_key','value':'$(base64 /path/to/your_private_key.pem)'}]"
+$ oc patch secrets/fluentd --type=json --patch "[{'op':'add','path':'/data/your_ca_cert','value':'$(base64 /path/to/your_ca_cert.pem)'}]"
+$ oc patch secrets/fluentd --type=json --patch "[{'op':'add','path':'/data/your_private_key','value':'$(base64 /path/to/your_private_key.pem)'}]"
```

Comment:
@jcantrill These commands need to be updated for the new secure-forward fields?

Reply:
This whole section should be updated for 4.3 to reflect the changes added in [1].  We need to note this applies to only using the 'legacy' way of LF to a fluent instance.  The intention is this method is replaced by the LF feature.  The requirements:
* Create a `secure-forward` configmap with content like you have identified
* If needed, create a secret named `secure-forward` and it will be mounted into the location defined in [1] 

@sichvoge I presume we can identify this version of secure forward as 'deprecated'?

[1] https://github.com/openshift/cluster-logging-operator/pull/325

Reply:
@jcantrill Does the user need to create a free-standing `secure-forward` configmap? Or is there still a `secure-forward.conf` section in the `fluentd` configmap?

Reply:
> @jcantrill Does the user need to create a free-standing `secure-forward` configmap? Or is there still a `secure-forward.conf` section in the `fluentd` configmap?

User creates the configmap `secure-forward` with a file named `secure-forward.conf`

If a secret is needed the user creates a secret named `secure-forward` with the keys they need which will be mounted at the path defined in the referenced PR